statistics-production/ees.qmd

```diff
@@ -150,6 +150,27 @@ See below for a diagram of the responsibilities of each role as part of the publ
 
 ![](../images/EES_roles.svg)
 
+---
+
+### Managing permissions
+
+---
+
+If you are a [publication owner](#roles-and-permissions) then you can add other analysts in your team to specific releases to allow them to edit and contribute to the release. 
+
+To do this, select your publication from the admin dashboard:
+
+![](../images/EES-admin-dashboard.PNG)
+
+Then go to the 'Team access' tab:
+
+![](../images/ees_team_access.png)
+
+In here you can see everyone who currently has access, and invite your team to contribute to the release, as well as manage and remove access. The release must already exist, and you'll need to give access for each new release you do in the publication.
```

Comment:
Maybe a bit pedantic but do we need a bit here about what happens if someone in the team is new to EES and needs to be added?

Reply:
It still works the same but good point - it's worth making this clear as I've seen people get confused by it in the past

Reply:
Updated this now